src/app/router/index.tsx

```diff
@@ -101,10 +101,9 @@ export const MainRouter = (): JSX.Element => {
         <AppBar position="sticky" color={'secondary'} elevation={0}>
             <Toolbar>
                 <div>
-                    {'We are changing our name to Brightlayer UI! Learn '}
+                    {'We are now Brightlayer UI! Learn '}
                     <a
                         style={{
-                            color: theme.palette.getContrastText(theme.palette.secondary.main),
```

Comment:
What will this look like for a blue secondary app bar?

Reply:
![Screen Shot 2021-12-20 at 12 20 47 PM](https://user-images.githubusercontent.com/13989985/146806919-e0f6c51e-be85-45fc-912d-226fc4232460.png)